.github/workflows/pr-workflow.yml

```
@@ -44,11 +50,73 @@ jobs:
+          ENABLE_FARGATE: false
+          ENABLE_DYNAMO: false
+          ENABLE_LAMBDA: true
+        run: |
+          npm i -g aws-cdk @tailwindcss/cli
+          make build
+
+  main-api:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)